.github/workflows/integration.yaml

```diff
@@ -0,0 +1,26 @@
+name: ops Integration Tests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '42 7 25 * *'
+  # Temporary, don't merge the PR trigger
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        preset: ['microk8s']  # ['k8s'] https://github.com/jnsgruk/concierge/issues/51
+        test-group: [1,2,3,4]
+
+    steps:
+      - run: sudo snap install --classic concierge
+      - run: sudo concierge prepare --juju-channel=3/stable --charmcraft-channel=3.x/stable -p "${{ matrix.preset }}"
+      - uses: actions/checkout@v4
+      - uses: astral-sh/setup-uv@v5
+      # Note that the test group count must match the groups, and there must be enough tests.
+      - run: uvx tox -e integration -- --test-group-count=4 --test-group=${{ matrix.test-group }}
```

Comment:
I don't think this test grouping thing works very well if we're using module-scoped fixtures (as there'd be a lot of repeated work). Personally I'd prefer to see tests split up across multiple files and have it more explicitly done by filename -- would be easier to see what's failed and diagnose issues.

Reply:
I'm specifically not using module-scoped fixtures.
This pytest plugin actually has a setting to group tests by files.
I'm not using it here, because my tests are independent and they are meant to be independent: they test different deployment options.